src/ex6/client/src/components/Button/Button.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import "monday-ui-react-core/dist/main.css";
+import "../Header/Header.css";
+
+const Button = ({ children, ...props }) => {
+  switch (props.as) {
+    case "input":
+      return (
+        <input
+          type="checkbox"
```

Comment:
I think it's better to seperate checkbox from this component.
button is in order to trigger action while checkbox is more like user's input.